pyccel/stdlib/ndarrays/ndarrays.h

```diff
@@ -10,6 +10,9 @@
 # include <stdbool.h>
 # include <stdint.h>
 
+#ifdef __cplusplus
```

Comment:
what is this?

Reply:
yeah this can be removed it was a previous fix that is now handled by the `__NVCC__` macro